src/chttpd/src/chttpd.erl

```diff
@@ -132,19 +185,21 @@ start_link(Name, Options) ->
 
     set_auth_handlers(),
 
-    Options1 = Options ++ [
-        {loop, fun ?MODULE:handle_request/1},
-        {name, Name},
-        {ip, IP}
-    ],
+    Options1 =
+        Options ++
+            [
+                {loop, fun ?MODULE:handle_request/1},
+                {name, Name},
+                {ip, IP}
+            ],
     ServerOpts = get_server_options("chttpd"),
     Options2 = merge_server_options(Options1, ServerOpts),
     case mochiweb_http:start(Options2) of
-    {ok, Pid} ->
-        {ok, Pid};
-    {error, Reason} ->
-        io:format("Failure to start Mochiweb: ~s~n", [Reason]),
-        {error, Reason}
+        {ok, Pid} ->
```

Comment:
+2 to always indent clauses in case even though CouchDB historically doesn't do it.

Reply:
+1

Reply:
+3